samples/nrf_desktop/src/modules/config.c

```diff
@@ -208,6 +207,13 @@ static bool event_handler(const struct event_header *eh)
 		if (is_config_event(eh)) {
 			struct config_event *event = cast_config_event(eh);
 
+			/* Do not store events that will be forwarded */
+			if (IS_ENABLED(CONFIG_BOARD_NRF52840_PCA10059)) {
```

Comment:
Well, this is common code for all devices and we have to check if we are currently running forwarder (dongle) or end recipient (mouse/keyboard). But instead I can check for BT_CENTRAL instead of specific dongle board.

Reply:
yes. BT_CENTRAL can probably be used.